server/plugin/webhook.go

```diff
@@ -88,6 +88,9 @@ func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if config.EnableWebhookEventLogging {
+		p.API.LogInfo("Webhook Event Log", "event", event)
```

Comment:
Why does this log to info and not debug?

Reply:
wasn't sure , ll change to debug